FireML/FireClassifier/networking.py

```diff
@@ -31,12 +37,18 @@ async def handler(reader: asyncio.StreamReader,
     # decrypt image
     data = AES.decrypt(data)
 
-    # TODO verify checksum
+    # Verify checksum
+    if len(data) != checksum:
+        logger.error(f'Checksum mismatch: expected {checksum}, got {len(data)}')
+        return
+    else:
+        logger.debug(f'Checksum match: {checksum}')
```

Comment:
Don't log just checksum. Take into account multiple clients accessing this service. Therefore is no way to trace who uploaded the file